pkg/fs/fs.go

```diff
@@ -701,7 +701,7 @@ func (fs *FileSystem) resolve(ctx meta.Context, p string, followLastSymlink bool
 			parent = inode
 			break
 		}
-		if i > 0 {
+		if parent > 1 {
```

Comment:
`i > 0` already skips the check for root directory?
What's the case (p = ?) are you trying to fix?